docs/source/theory/boundaryConditions.rst

```diff
@@ -727,6 +727,28 @@ important for stability for CVFEM tet-based meshes where a natural
 non-orthogonality exists between the boundary and interior integration
 point.
 
+In rare use cases, the usage of the standard open boundary mass flow 
+rate expression, which includes pressure contributions, is not appropriate
+due to complex temperature/buoyancy specifications, e.g., what is commonly
+seen in Bousinesq Atmospheric Boundary Layer (ABL), simulations. In these cases, 
```

Comment:
sp: Boussinesq. Why is this specific to Boussinesq ABL and not for, say, Anelastic approximation?

Reply:
Probably can. By e.g., I meant a specific case, not all that might be around. I can change if you like...